src/org/labkey/test/components/ui/ontology/ConceptInfoTabs.java

```diff
@@ -71,6 +72,8 @@ public ConceptInfoTabs showPathInformation()
     public String getTitle()
```

Comment:
If no concept is selected, the 'none selected' element will be present.  surfacing its text in `getTitle()` in this case makes sense, I think (especially because it handles the case where the title element isn't present)